src/__tests__/database.http.spec.ts

```diff
@@ -0,0 +1,86 @@
+import nock from "nock";
+import { Tigris } from "../../dist/tigris";
+import { Database } from "../../dist/database/database.interface";
+
+describe("Database", () => {
+  let tigrisClient: Tigris;
+  let db: Database;
+
+  beforeEach(async () => {
+    tigrisClient = new Tigris({
+      serverUrl: "http://localhost:8081",
+      projectName: "test",
+    });
+    await tigrisClient.initialize();
+    await tigrisClient.getDatabase().initializeBranch();
```

Comment:
Switch this and the `db` assignment below so that we only run `tigrisClient.getDatabase()` once.